<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>918</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1876</width>
         <height>843</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>7</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Example Queries</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="midLineWidth">
               <number>3</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Find the 5</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="heavyRad">
                <property name="text">
                 <string>Heaviest</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="lightRad">
                <property name="text">
                 <string>Lightest</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Pokemon</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="weightButton">
              <property name="text">
               <string>Return</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="midLineWidth">
               <number>3</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Return 5 Pokemon with the</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="highRad">
                <property name="text">
                 <string>highest</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="lowRad">
                <property name="text">
                 <string>lowest</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>stats</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="statsButton">
              <property name="text">
               <string>Return</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="midLineWidth">
               <number>3</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Type with</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="mostRad">
                <property name="text">
                 <string>most</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="leastRad">
                <property name="text">
                 <string>least</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>number of moves</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="numMovesPerType">
              <property name="text">
               <string>Return</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_4">
              <property name="midLineWidth">
               <number>3</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Find the number of moves of Pokemon with its name starting with the letter</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>can learn in each version</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="startingLetterBtn">
                <property name="text">
                 <string>Return</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_5">
              <property name="midLineWidth">
               <number>3</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Find all legendary Pokemon that can evolve</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="legendEvo">
              <property name="text">
               <string>Return</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_6">
              <property name="midLineWidth">
               <number>3</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Return all Pokemon with more than 2 potential abilities.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="greaterThanTwo">
              <property name="text">
               <string>Return</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_7">
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="midLineWidth">
               <number>3</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Return the distribution of Pokemon Types in each region</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pokemonTypeDist">
              <property name="text">
               <string>Return</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_8">
              <property name="midLineWidth">
               <number>3</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Return all Pokemon with Palindrome names</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="palindromeNames">
              <property name="text">
               <string>Return</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_9">
              <property name="midLineWidth">
               <number>3</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Return all Pokemon that are not part of an evolution line and are not mythical/legendary</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="noEvoBtn">
              <property name="text">
               <string>Pokemon w/ no evo line</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_10">
              <property name="midLineWidth">
               <number>3</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Return the average height of all egg groups</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="avgHeightEggGroup">
              <property name="text">
               <string>Average height of all egg groups</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_11">
              <property name="midLineWidth">
               <number>3</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="tableVertical">
            <item>
             <widget class="QScrollArea" name="scrollArea_2">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_2">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>1113</width>
                 <height>815</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="1" column="0">
                 <widget class="QTableWidget" name="dbTable"/>
                </item>
                <item row="0" column="0">
                 <widget class="QLineEdit" name="tableName">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="lineEdit_2">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Dump Tables</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pokemon">
              <property name="text">
               <string>Pokemon</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="generation">
              <property name="text">
               <string>Generation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ability">
              <property name="text">
               <string>Ability</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="forms">
              <property name="text">
               <string>Forms</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="types">
              <property name="text">
               <string>Types</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="typeDamage">
              <property name="text">
               <string>Type Damage</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="moves">
              <property name="text">
               <string>Moves</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="version">
              <property name="text">
               <string>Version</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="versionGroup">
              <property name="text">
               <string>Version Groups</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="items">
              <property name="text">
               <string>Items</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="regularEvo">
              <property name="text">
               <string>Regular Evolution</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="uniqueEvo">
              <property name="text">
               <string>Unique Evolution</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="encounterSlots">
              <property name="text">
               <string>Encounter Slots</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="encounterMethod">
              <property name="text">
               <string>Encounter Method</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="regions">
              <property name="text">
               <string>Regions</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="locations">
              <property name="text">
               <string>Locations</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="areas">
              <property name="text">
               <string>Areas</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="eggGroups">
              <property name="text">
               <string>Egg Groups</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="learnMethod">
              <property name="text">
               <string>Learn Methods</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pkmnMoveVrsnLrn">
              <property name="text">
               <string>Pokemon Move Version Learn</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pokemonForms">
              <property name="text">
               <string>Pokemon With Forms</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pokemonTypes">
              <property name="text">
               <string>Pokemon Types</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="formTypes">
              <property name="text">
               <string>Form Types</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pkmnHeldItemVrsn">
              <property name="text">
               <string>Pokemon Held Item Version</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="machines">
              <property name="text">
               <string>Machines</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pkmnAreaVrsnEnc">
              <property name="text">
               <string>Pokemon Area Version Encounter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pokemonAbilities">
              <property name="text">
               <string>Pokemon Abilities</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pokemonEggGroups">
              <property name="text">
               <string>Pokemon Egg Groups</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
